config/locales/en.yml

```diff
@@ -225,6 +225,9 @@ en:
       transparency:
         one: Transparency data
         other: Transparency data
+      world_location_news_article:
```

Comment:
Hi,

@fofr we can chat about this in the morning, but I have added a commit which translates 'world location news article' to 'news article' in English. 

Reason for this was that I was going to pull in the `document_type` as opposed to the `format` in `_history_notice` which would mean it would pull in `"document_type": "news_article"` for WLNAs. Sadly this messed up the text for a few other formats, for example in Detailed Guides the text “This guidance” changed to “This detailed guide".

Could have added a condition to the view that pulled in the `document_type` for `WLNA`s and the `format` for the other doc_types but thought it might be better to keep it consistent and keep the blank lines in the translation files until we get a translation at some point.